### PR TITLE
Foxicide fluff items, minor material weapon refactor

### PR DESCRIPTION
**Pull request by cebutris**
http://yawn-widerstation.proboards.com/thread/9/fluff-items?page=6&scrollTo=1384

Direct any questions to Foxicide, not me!

EDIT: Also, this adds a var to materials weapons that sets whether or not they're named based on the matieral, which is defaulted to yes. Should be useful if any other custom material items come around.

### CODE REVIEW

**Comment by Repede** [comment]
Need a modkit from plasteel knife to their knife, and also there needs the actual loadout path linked to their ckey and character or they will never spawn with it :octocat:

**Comment by cebutris** [comment]
Shit I knew I forgot something

File: code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm [hunk 1]
```diff
@@ -346,6 +346,22 @@
 
 //  F CKEYS
 
+//Foxicide
+//Kaith
+/datum/gear/fluff/kaith_knife
+	path = /obj/item/weapon/kitchenknife/tacknife/unathiknife/fluff/kaith
```
Comment (Repede) [inline code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm]: needs to be a modkit, from plasteel survival knife to that path. Should have some modkit examples in custom items and such files.

**Comment by Hawk-v3** [comment]
Pending changes, merge on hold.

File: code/modules/vore/fluffstuff/custom_clothes_yw.dm [hunk 2]
```diff
@@ -1591,3 +1591,12 @@
 	icon_override = 'icons/vore/custom_clothes_yw.dmi'
 	item_state = "valormedal1_mob"
 	overlay_state = "valormedal1_mob"
+
+// ******
+// Foxicide
+// ******
+/obj/item/weapon/storage/backpack/fluff/kaith
+	name = "Light Duffel bag"
```
Comment (Repede) [inline code/modules/vore/fluffstuff/custom_clothes_yw.dm]: Duffle*

File: code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm [hunk 3]
```diff
@@ -346,6 +346,22 @@
...
+	display_name = "Trusty Survival Knife"
+	description = "An old looking knife with an outdated wooden handle. Still looks robust, though."
+	ckeywhitelist = list("foxicide")
+	character_name = list("Kaith")
+
+/datum/gear/fluff/kaith_bag
+	path = /obj/item/weapon/storage/backpack/fluff/kaith
+	display_name = "Light Duffel bag"
```
Comment (Repede) [inline code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm]: Duffle*

**Comment by cebutris** [comment]
I'll fix it tommorow! The local repo was on my laptop which has been having charger issues but I just got a new one so I can get this rolled out properly

**Comment by cebutris** [comment]
Hey remember when I said I would do the thing and then I didn't? That was fun.

I did the thing now, though. It's a modkit, and it's tested

**Pull request merged**